dune

```diff
@@ -1,7 +1,10 @@
 ; Default flags for all Coq libraries.
+
 (env
- (dev     (flags :standard -rectypes -w -9-27-50+60))
- (release (flags :standard -rectypes)))
+ (dev     (flags :standard -rectypes -w -9-27-50+40+60))
+ (release (flags :standard -rectypes)
+          (ocamlopt_flags -O3 -unbox-closures))
```

Comment:
Would this work work if coq is installed via opam? `release` is the profile that will be selected for all switches, not just those that use flambda.

We don't have a great way of addressing this in dune unfortunately as there's no way to conditionally include flags in env.